tests/integration/guzzle6/tests/bootstrap.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
```

Comment:
If this is the only content, maybe you can just specify the autoload.php in the phpunit config file

Reply:
done.